source/extensions/compression/gzip/decompressor/zlib_decompressor_impl.cc

```diff
@@ -17,27 +17,17 @@ namespace Compression {
 namespace Gzip {
 namespace Decompressor {
 
-namespace {
-
-// How many times the output buffer is allowed to be bigger than the size of
-// accumulated input. This value is used to detect compression bombs.
-// TODO(rojkov): Re-design the Decompressor interface to handle compression
-// bombs gracefully instead of this quick solution.
-constexpr uint64_t MaxInflateRatio = 100;
-
-} // namespace
-
 ZlibDecompressorImpl::ZlibDecompressorImpl(Stats::Scope& scope, const std::string& stats_prefix)
-    : ZlibDecompressorImpl(scope, stats_prefix, 4096) {}
+    : ZlibDecompressorImpl(scope, stats_prefix, 4096, 100) {}
```

Comment:
I'm not sure if this c'tor is used anywhere outside of tests, but if not, should it be removed (to ensure single point of definition of the default values in `config.cc`)?

Reply:
I have checked that it only be used in tests, and it's good to remove it.